src/services/dashboards/widgets/cost-map/CostMapWidget.vue

```diff
@@ -89,29 +99,51 @@ const drawChart = (chartData) => {
         nodePaddingInner: 4,
     };
     const series = createTreeMapSeries(seriesSettings);
-    series.rectangles.template.setAll({
-        strokeWidth: 1,
-    });
-
-    setChartColors(series, state.colorSet);
-    // Todo: add gradient background color
-    // const themeColorName: WidgetTheme = props.theme ? props.theme : 'violet';
-    // series.rectangles.template.adapters.add('fill', (value, target) => {
-    //     const dataItem = target.dataItem;
-    //     if (dataItem?.dataContext?.[VALUE_FIELD_NAME] === 'Root') {
-    //         return color(palette.white);
-    //     }
-    //     if (dataItem?.dataContext?.[CATEGORY_FIELD_NAME]) {
-    //         return color(palette[themeColorName][400]);
-    //     }
-    //     return color(palette[themeColorName][400]);
-    // });
+
+    const getConvertedChartData = (rawData): CostMapData[] => {
```

Comment:
suggestion: I think this function is only related to color! How about to change func name?🤔 like...setColorData..? getChartDataWithColor...?

Reply:
I think 'setColorData' is good. Because other widgets also use a function with that name when changing colors